Code/autopkglib/MunkiInstallsItemsCreator.py

```diff
@@ -100,7 +105,7 @@ def create_installs_items(self):
             for item in installs_array:
                 cmp_key = None
                 # If it's a string, set it for all installs items
-                if isinstance(self.env["version_comparison_key"], basestring):
+                if isinstance(self.env["version_comparison_key"], str):
```

Comment:
As in the other PR, this changes behavior in Python 2.

Reply:
I'm going to punt this down the line until better testing can be developed.